assets/js/theme/global/mobile-menu-toggle.js

```diff
@@ -53,9 +56,14 @@ export class MobileMenuToggle {
         return this.$menu.hasClass('is-open');
     }
 
+    get subMenuIsOpen() {
```

Comment:
Also we just use it in once place. Can we just add the check directly instead of a separate method ?